src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/Delegation/DelegatedCompletionListProviderTest.cs

```diff
@@ -103,7 +103,7 @@ await _provider.GetCompletionListAsync(
     }
 
     [Fact]
-    public async Task HtmlDelegation_UnsupportedTriggerCharacter_TranslatesToInvoked()
+    public async Task HtmlDelegation_UnsupportedTriggerCharacter_ReturnsNull()
```

Comment:
It's interesting we had a test for this specific behavior... I don't know if this change would break anything, though I can't see how sending explicit completion invokes to HTML LSP for something that's not an HTML trigger character is a good thing. VS HTML at least behaves decently by returning nothing in inappropriate situation, but VS Code HTML is very aggressive with low quality matches ("better return anything than nothing"?) and that makes typing very annoying. In VS, it seems like a wasted message sent to HTML LSP.